### PR TITLE
Fix unsigned integer overflow

### PR DESCRIPTION
**Pull request by tuoxie007**
## Changes in this pull request

Issue fixed: #

### Checklist

- [x] All tests pass. Demo project builds and runs.
- [ ] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by lorixx** [comment]
ummm @tuoxie007 r u still merging this PR? thanks

**Comment by lorixx** [comment]
@tuoxie007 can you sign the CLA? thanks

**Comment by lorixx** [comment]
@tuoxie007 can you sign the CLA? thanks

**Comment by tuoxie007** [comment]
> @tuoxie007 can you sign the CLA? thanks

Sorry for late, done for cla.

**Comment by lorixx** [comment]
Thank you @tuoxie007 Jason, can you rebase and update the PR w/ the latest master? thx

File: Source/Common/IGListBatchUpdateData.mm [hunk 1]
```diff
@@ -33,6 +33,7 @@ + (void)_cleanIndexPathsWithMap:(const std::unordered_map<NSInteger, IGListMoveI
                     indexPaths:(NSMutableArray<NSIndexPath *> *)indexPaths
                        deletes:(NSMutableIndexSet *)deletes
                        inserts:(NSMutableIndexSet *)inserts {
+    if (indexPaths.count == 0) return;
```
Comment (lorixx) [inline Source/Common/IGListBatchUpdateData.mm]: Let's use {} :))

```
if (indexPaths.count == 0) {
    return;
}
```

Comment (tuoxie007) [reply Source/Common/IGListBatchUpdateData.mm]: done

**Pull request closed without merging**